src/encoded/static/components/matrix_target.js

```diff
@@ -105,7 +105,7 @@ const getTargetData = (context, assayTitle, organismName) => {
         targetData[subTab] = { headerRow, dataRow, assayTitle, organismName };
     });
 
-    const subTabs = subTabList;
+    const subTabs = subTabList.sort();
```

Comment:
Is it OK that `subTabs` and `subTabsHeaders` will have a different order? I can’t quite tell.

Reply:
subTabsHeaders is redundant and was removed. It is created only where it is needed and map is used. Javascript map gurantees order ([https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Objects_and_maps_compared](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map#Objects_and_maps_compared))